qa/rpc-tests/dip4-coinbasemerkleroots.py

```diff
@@ -81,6 +81,79 @@ def run_test(self):
         # When comparing genesis and best block, we shouldn't see the previously added and then deleted MN
         mnList = self.test_getmnlistdiff(null_hash, self.nodes[0].getbestblockhash(), {}, [], expectedUpdated2)
 
+        #############################
+        # Now start testing quorum commitment merkle roots
+
+        height = self.nodes[0].getblockcount()
+        # Test DIP8 activation once with a pre-existing quorum and once without (we don't know in which order it will activate on mainnet)
+        self.test_dip8_quorum_merkle_root_activation(True)
+        for n in self.nodes:
+            n.invalidateblock(n.getblockhash(height + 1))
+        first_quorum = self.test_dip8_quorum_merkle_root_activation(False)
```

Comment:
Rewinding this many blocks is a heavy operation, make sure all nodes are on the same page before moving any further.
```suggestion
        self.sync_all()
        first_quorum = self.test_dip8_quorum_merkle_root_activation(False)
```